server/app/services/export/PdfExporter.java

```diff
@@ -70,15 +74,23 @@ public InMemoryPdf export(Application application) throws DocumentException, IOE
             answers,
             applicantNameWithApplicationId,
             application.getProgram().getProgramDefinition(),
-            application.getLatestStatus());
+            application.getLatestStatus(),
+            getSubmitTime(application.getSubmitTime()));
     return new InMemoryPdf(bytes, filename);
   }
 
+  private String getSubmitTime(Instant submitTime) {
+    return submitTime == null
+        ? "Application submitted without submission time marked."
```

Comment:
That sentence isn't really clear. When `application.getSubmitTime()` is null that means the application has not been submitted. I would just say "Application has not been submitted".

Reply:
I don't think an application which hasn't been submitted will ever appear here. It is possible that application is no not in active state yet or work case, the submit time didn't get recorded due to some weird reason.

Reply:
The java docs for the function which fetches the application list-   
https://github.com/civiform/civiform/blob/0513cda4ecf02d3227d9357ed90a01b318bf4990/server/app/repository/ProgramRepository.java#L237 
   * Get all submitted applications for this program and all other previous and future versions of
   * it where the application matches the specified filters. Does not include drafts or deleted
   * applications. Results returned in reverse order that the applications were created.